.github/workflows/build-assets-on-release.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     tags:
       - '*.*.*'
+      - '!*.*.*-alpha'
```

Comment:
This will trigger the build if it's not tagged with `-alpha` (the not at the beginning), that's intentional?